### PR TITLE
Convert protocol elements to WebdriverIO elements

### PR DESCRIPTION
**Pull request by christian-bromann**
**Is your feature request related to a problem? Please describe.**
There are some commands like `getActiveElement` that returns something like `element-6066-11e4-a52e-4f735466cecf:"ELEMENT-57"`

There is no way to interact with such object like with WebdriverIO element

**Goal**
I'd like to be able to wrap such responses, ideally automatically somehow

**Suggested solutions**
- add command WebdriverIO for every protocol command, like `getActiveElement` to wrap response with WebdriverIO element;
- add some command, like `browser.wrap(browser.getActiveElement())` to be able to convert protocol element objects to WebdriverIO elements
- automatically convert responses that have only `element-6066-11e4-a52e-4f735466cecf` in body to WebdriverIO elements.

### CODE REVIEW

**Comment by christian-bromann** [comment]
@webdriverio/project-committers what do you all think about this solution?

**Comment by CrispusDH** [comment]
I think this solution is good enough for such use case.

File: packages/webdriverio/src/commands/browser/$.js [hunk 1]
```diff
@@ -32,18 +34,32 @@
         });
         console.log(text.$$('li')[2].$('a').getText()); // outputs: "API"
     });
+
+    it('should allow to convert protocol result of an element into a WebdriverIO element', () => {
+        const activeElement = browser.getActiveElement();
+        console.log($(activeElement).getTagName); // outputs active element
```
Comment (mgrybyk) [inline packages/webdriverio/src/commands/browser/$.js]: .getTagName should be a function call in all occurrences

**Comment by mgrybyk** [comment]
Not sure why but I can't neither approve nor request changes for PRs both from mobile and desktop, that's why leaving comments only

**Pull request merged**